### DISCUSSION TITLE
Backup of game world?

### DISCUSSION THREAD

**Discussion by mirkster** [AlexProgrammerDE/balena-minecraft-server#90] [Q&A] [answered]
Is there anyway to automate backups of the game world? I am concerned about a corrupted SD card in the future. I know that I can login with WinSCP and just download the world from serverfiles but was looking for a way to setup a regular auotmated backup either locally or preferably to my Synology.

**Comment by AlexProgrammerDE** [comment, marked as answer]
> 
> 
> Is there anyway to automate backups of the game world? I am concerned about a corrupted SD card in the future. I know that I can login with WinSCP and just download the world from serverfiles but was looking for a way to setup a regular auotmated backup either locally or preferably to my Synology.

Maybe this could help you to get started? https://www.synology.com/en-nz/knowledgebase/DSM/tutorial/Backup/How_to_back_up_Linux_computer_to_Synology_NAS
I've never been into nas and backuping, but let me know if you need help to get something running for that. 👍🏻

**Reply by mirkster** [reply]
Thanks that gave me an idea of using rsync but I didn't set it up like they suggested by pushing the backup to the Synology, I wanted to pull the backup from the Minecraft server. So I installed rsync on to scp-server and then configured the rsync daemon. Then I created a backup task on Active Backup for Business on the Synology to connect to the rsync daemon and backup the files to a folder on the Synology.
